packages/esm-patient-medications-app/translations/en.json

```diff
@@ -16,6 +16,8 @@
   "dispensingInformation": "3. Dispensing Information",
   "dosageInstructions": "1. Dosage Instructions",
   "dose": "Dose",
+  "drugSearchField": "Search Field",
+  "drugSearchFieldPlaceholder": "Search for a drug or orderset (e.g. \"Aspirin\")",
```

Comment:
@reagan-meant, please do update the translation file for Khmer as well for the new keys and delete key.
CC: @rbuisson!

Reply:
@reagan-meant, do push the updated file in the same branch.
Thanks!

Reply:
@vasharma05 the khmer translations will be translated, I am putting the text back as it was since we solved the issue that was causing the translation not to show.

Reply:
Actually the translation file only takes the default value from the `t(key, defaultValue)`. The library doesn't do translation by itself. The translations other than `en` is done manually.